Decred Wallet/Features/Seed Backup/BackupReminderViewController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  BackupReminderViewController.swift
+//  Decred Wallet
+//
+// Copyright (c) 2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+import UIKit
+import Dcrlibwallet
+import SwiftRichString
```

Comment:
I'm not particularly sure of this new pod yet. It's just 1 screen that it's being used for, I believe we can implement this screen's design without an external library.